src/extension.ts

```diff
@@ -157,6 +158,10 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
 			}
 		});
 
+		vscode.commands.registerCommand('camelk.integrations.createNewIntegrationFile', async() => {
```

Comment:
Optional: 
You can simply reference a function here instead of writing an async anonymous arrow function. 
Example: 
`context.subscriptions.push(vscode.commands.registerCommand('datavirt.create.vdb', createVDBCommand));`